commons/src/main/java/com/pryv/api/OnlineManager.java

```diff
@@ -26,17 +26,19 @@
 import okhttp3.Response;
 
 /**
- * OnlineEventsAndStreamsManager manages data from online Pryv API
+ * OnlineManager manages data from online Pryv API
  *
  * @author ik
  */
-public class OnlineEventsAndStreamsManager {
+public class OnlineManager {
```

Comment:
I know this class will be replaced, but why the refactor?

Reply:
It makes no more sense to contain Events/Streams since the goal is to have an online manager that handle any API resources in a generic manner (as it already did for Accesses). I have still hesitations to drop this class or not but in any case, I want it to be light and delegate as much work as possible to Connection objects.

Reply:
OK, well if you drop the class as you did for Accesses, I don't care what you call it, although it does pollutes the commit diff if you change it for no reason.
If you keep the class, I would be glad if you found a better name, for example "ApiCaller"? I was uninspired at the point of the naming, and it was to go in pair with "CacheManager", but I generally wouldn't recommend naming anything "Handler" or "Manager".
Anyway it isn't worth spending much time on this, just sharing my point of view.